py3status/modules/process_status.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display if a process is running.
+Display status of a given process.
```

Comment:
Again I think the old description is better `Display if a process is running.`  status implies running/zombie niceness etc